### PR TITLE
Add FixAll support for UseExpressionBody/BlockBody refactoring

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

File: src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs [hunk 1]
```diff
@@ -152,8 +185,85 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
...
+                declarationsToFix = declarationsToFix.Reverse();
+
+                // Track all the declaration nodes to be fixed so we can get the latest declaration node in the current root during updates.
+                var currentRoot = root.TrackNodes(declarationsToFix);
+
+                foreach (var declaration in declarationsToFix)
+                {
+                    // Get the current document, root, semanticModel and declaration.
+                    document = document.WithSyntaxRoot(currentRoot);
+                    currentRoot = await document.GetRequiredSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
```
Comment (genlu) [inline src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs:256]: Is this necessary?

**Review by genlu — APPROVED** [review]

File: src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs (hunk 1 above)
Comment (mavasani) [reply src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs:256]: Yes, we eventually end up reaching [here](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Syntax/CSharpSyntaxTree.cs,330), which creates a new `ParsedSyntaxTree` with `cloneRoot: true`, leading to the tree being created with a cloned root [here](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Syntax/CSharpSyntaxTree.ParsedSyntaxTree.cs,51).

**Comment by mavasani** [comment]
Thank you for the review @genlu!

**Comment by jnm2** [comment]
Closes https://github.com/dotnet/roslyn/issues/41791.

**Pull request merged**